getting-started/templates/systemlink-values.yaml

```diff
@@ -55,6 +55,9 @@ global:
   ##
   telemetry:
     ## Specify whether to collect telemetry data.
+    ## Requires deployment of an OpenTelemetry collector and Prometheus.
+    ## OpenTelemetry: https://opentelemetry.io/docs/what-is-opentelemetry/
```

Comment:
Technically, the OpenTelemetry line should go with `openTelemetryExporterOtlpEndpoint`. The `global.telemetry.enabled` by itself without setting `openTelemetryExporterOtlpEndpoint` is expected to only use Prometheus.